File: C++/Easy/3178. Find the Child Who Has the Ball After K Seconds.cpp
// https://leetcode.com/problems/find-the-child-who-has-the-ball-after-k-seconds/description/

class Solution {
public:
    int numberOfChild(int n, int k) {
        k = k % (2 * n - 2);
        return (k < n) ? k : 2 * n - k - 2;
    }
};

File: Python/Easy/3178. Find the Child Who Has the Ball After K Seconds.py
# https://leetcode.com/problems/find-the-child-who-has-the-ball-after-k-seconds/description/

class Solution:
    def numberOfChild(self, n: int, k: int) -> int:
        k = k % (2 * n - 2)
        return k if k < n else 2 * n - k - 2 